src/components/PrisonProfilePage.js

```diff
@@ -0,0 +1,58 @@
+import { render } from "react-dom";
+
+class MainContainer extends React.Component {
+  render() {
+    return (
+      <div className="main-container">
+        <div className="nav-container">
+          <a></a>
+          <a></a>
+          <a></a>
+          <a></a>
+          <a></a>
+        </div>
+
+        <aside className="group-name">
```

Comment:
We don't need to hard code any `html` tags.
Create two `divs`, the left div should take the prison info data from our `State` and display them here.
The right div should contain a map function to take the inmate info and display them.